src/muse/readers/csv.py

```diff
@@ -816,10 +816,19 @@ def expand_paths(path):
     datas = {}
     for path in allfiles:
         data = pd.read_csv(path, low_memory=False)
+        assert all(u in data.columns for u in indices)
```

Comment:
Note that `assert` statements aren't included in optimised Python bytecode. But I guess if one of these columns is missing an error will be raised below?